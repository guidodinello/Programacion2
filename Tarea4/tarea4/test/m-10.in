# 
insertarEnBinario (3,3.3)
insertarEnBinario (6,6.6)
insertarEnBinario (2,2.2)
insertarEnBinario (7,7.7)
insertarEnBinario (9,9.9)
insertarEnBinario (8,8.8)
linealizacion
recorridaPorNiveles
binarioBalanceado 6 10 20 30 40 50 60
linealizacion
recorridaPorNiveles
enAlguno 5 1 3 7 8 9
soloEnA 5 1 3 7 8 9
reiniciar
agregarAIterador 1
agregarAIterador 2
agregarAIterador 5
agregarAIterador 8
enAlguno 5 1 3 7 8 9
soloEnA 5 1 3 7 8 9
Fin

