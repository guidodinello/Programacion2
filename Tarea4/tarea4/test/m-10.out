1>#  .
2>Insertado.
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>Insertado.
8>(2,2.20)(3,3.30)(6,6.60)(7,7.70)(8,8.80)(9,9.90)
9>8 | 9 | 7 | 2 6 | 3 
10>
11>(10,0.00)(20,0.00)(30,0.00)(40,0.00)(50,0.00)(60,0.00)
12>20 40 60 | 10 50 | 30 
13>1 3 7 8 9 
14>
15>Estructuras reiniciadas.
16>Agregando.
17>Agregando.
18>Agregando.
19>Agregando.
20>1 2 3 5 7 8 9 
21>2 5 
22>Fin.
